### PR TITLE
CI - Ability to run only e2e cloud in case is the only affected folder

### PR DESCRIPTION
**Pull request by mauriziovitale**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**
- remove the skip lint 
- ability to run specs
- affected e2e-cloud folder

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: scripts/affected-libs.sh [hunk 1]
```diff
@@ -58,7 +54,7 @@ if [ ! -f $DIRECTORY/deps.txt ]; then
     npm run affected:libs -- $HEAD_SHA_BRANCH "HEAD" > $DIRECTORY/deps.txt
 fi
 
-cat $DIRECTORY/deps.txt
+#cat $DIRECTORY/deps.txt
```
Comment (DenysVuika) [inline scripts/affected-libs.sh]: do we need that or can be removed?

File: scripts/test-e2e-lib.sh [hunk 2]
```diff
@@ -4,7 +4,7 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 cd "$DIR/../"
 BROWSER_RUN=false
 DEVELOPMENT=false
-EXECLINT=true
+EXECLINT=false
```
Comment (DenysVuika) [inline scripts/test-e2e-lib.sh]: we don't lint by default now?

Comment (mauriziovitale) [reply scripts/test-e2e-lib.sh]: nope

File: scripts/affected-libs.sh (hunk 1 above)
Comment (mauriziovitale) [reply scripts/affected-libs.sh]: it can be removed it was used on debugging time to make sure the file content was the expected one

**Pull request merged**